examples/discovery-search-app/cypress/support/utils/defaultHomepage.ts

```diff
@@ -0,0 +1,16 @@
+//TODO: break down the logic in this funtion so that these defaults can be overwritten. We'll probably want to
+// create more util files to make this functionality more intuitive and modular.
+export function mockDefaultHomePage() {
+  // Sets up and handles the collections, component settings, and initial query requests that run on page-load
+  cy.server();
+  cy.fixture('collections/collections.json').as('collectionsJSON');
+  cy.route('GET', '**/collections?version=2019-01-01', '@collectionsJSON').as('getCollections');
+  cy.fixture('component_settings/componentSettings.json').as('componentSettingsJSON');
+  cy.route('GET', '**/component_settings?version=2019-01-01', '@componentSettingsJSON').as(
+    'getComponentSettings'
+  );
+  cy.fixture('query/query.json').as('queryJSON');
+  cy.route('POST', '**/query?version=2019-01-01', '@queryJSON').as('postQuery');
+  cy.visit('/');
+  cy.wait(['@getCollections', '@getComponentSettings', '@postQuery']);
```

Comment:
Note to self: make it possible to pass in overrides for these fixtures. I'm thinking specifically of the facets tests, where we want to change the `componentSettings` response that comes in on page load.